### PR TITLE
Remove go version

### PR DESCRIPTION
**Pull request by belak**

### CODE REVIEW

**Comment by guswynn** [comment]
wow!

File: README.md [hunk 1]
```diff
@@ -4,42 +4,29 @@ Seabird has been an IRC bot for the last 10 years in many different
...
+implementation.
 
 ## Building
 
 The easiest way to build and deploy `seabird-core` is to use the [official
-docker images](https://hub.docker.com/r/seabirdchat/seabird-core). The `go` and
-`rust` tags point to the Go and Rust implementations respectively on the master
-branch. Additionally, the `latest` tag will point to whichever implementation is
-given focus when implementing. This is currently the `rust` version.
+docker images](https://hub.docker.com/r/seabirdchat/seabird-core).
```
Comment (jsvana) [inline README.md:16]: `docker image]`

File: README.md [hunk 2]
```diff
@@ -4,42 +4,29 @@ Seabird has been an IRC bot for the last 10 years in many different
...
 
 In order to build these, you can use one of the following:
 
 ```sh
-docker build -t seabird-core:go -f Dockerfile-go .
-docker build -t seabird-core:rust -f Dockerfile-rust .
+docker build -t seabird-core:rust .
```
Comment (jsvana) [inline README.md:21]: Should this still have the `:rust` suffix?

File: README.md [hunk 3]
```diff
@@ -4,42 +4,29 @@ Seabird has been an IRC bot for the last 10 years in many different
...
 ```
 
-If you want to run the server outside of docker, there is information in each
-implementation's sub-folder README.
+If you want to develop locally, simply use the normal cargo workflow in order to
+build/run seabird-core:
```
Comment (jsvana) [inline README.md:25]: Unrelated to this PR but might be worth adding a reminder to `git submodule update --init`

**Review by jsvana — APPROVED** [review]
LGTM!

**Pull request merged**